code/modules/hydroponics/grown/towercap.dm

```diff
@@ -1,3 +1,5 @@
+#define OUTDOOR_AREAS list(/area/f13/wasteland, /area/f13/desert, /area/f13/farm, /area/f13/forest, /area/f13/ruins)
```

Comment:
I'd replace this with a GLOBAL_LIST_INIT(outdoor_areas, list()). I can see this list being useful for other things too!